File: scripts/dog.js
// Description
//   Moss cat command
//   Why that wasn't there already I don't even know
//
// Commands:
//   `!dog` - Like !cat, but for dog people.

module.exports = function (robot) {
	robot.respond(/!?dog/i, function (res) {
		res.send("```\n" +
            "                                __\n" +
            "         ,                    ,\" .\`--o\n" +
            "        ((                   (  | __,\'\n" +
            "         \\\\~----------------' \\_;/\n" +
            "         (                      /\n" +
            "         /) ._______________.  )\n" +
            "        (( (               (( (         hjw\n" +
            "         \`\`-\'               \`\`-\'\n\n```");
  });
};

File: uqcsbot/scripts/dog.py
from uqcsbot import bot, Command


@bot.on_command("dog")
def handle_dog(command: Command):
    """
    `!dog` - Like !cat, but for dog people.
    """
    dog = "\n".join(("```",
                     "                          _         ",
                     "                       ,:'/   _..._ ",
                     "                      // ( `""-.._.'",
                     "                      \\| /    O\\___",
                     "                      |    O       4",
                     "                      |            /",
                     "                      \\_       .--' ",
                     "                      (_'---'`)     ",
                     "                      / `'---`()    ",
                     "                    ,'        |     ",
                     "    ,            .'`          |     ",
                     "    )\\       _.-'             ;     ",
                     "   / |    .'`   _            /      ",
                     " /` /   .'       '.        , |      ",
                     "/  /   /           \\   ;   | |      ",
                     "|  \\  |            |  .|   | |      ",
                     " \\  `\"|           /.-' |   | |      ",
                     "  '-..-\\       _.;.._  |   |.;-.    ",
                     "        \\    <`.._  )) |  .;-. ))   ",
                     "        (__.  `  ))-'  \\_    ))'    ",
                     "            `'--\"`  jgs  `\"\"\"`      ```"))

    bot.post_message(command.channel_id, dog)
